plugins/essif-lab_contactform7/admin/index.php

```diff
@@ -0,0 +1 @@
+<?php // Silence is golden
```

Comment:
Als je met classes werkt is dit niet echt nodig. Daarnaast wordt/is onze plugin code toch publiek dus kan men als ze willen dit toch wel achterhalen.

...dat er een `essif-lab_contactform7/admin/`  directory is